### PR TITLE
[multi-asic]: Add azure pipeline script to run kvm test

### PR DESCRIPTION
**Pull request by SuvarnaMeenakshi**
[multi-asic]: Add azure pipeline script to run kvm test
on multi-asic KVM testbed against 201911 multi-asic image.

Signed-off-by: SuvarnaMeenakshi <sumeenak@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X] Test case(new/improvement)

#### What is the motivation for this PR?
Add azure pipeline script to bring up multi-asic testbed and run kvm tests.

#### How did you do it?
Add azure pipeline script :
- Bring up multi-asic KVM testbed with 201911 image as 201911 image is currently stable for multi-asic
- Deploy minigraph
- Run multi-asic KVM tests

### CODE REVIEW

File: tests/pipelines/multi_asic_201911.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        sudo chown -R $username.$username /data/sonic-vm
+        pushd  /data/sonic-mgmt
+        git remote update
+        git reset --hard origin/master
+        docker rm -f sonic-mgmt-masic || true
+        ./setup-container.sh -n sonic-mgmt-masic -d /data
+        sed -i s/use_own_value/${username}/ ansible/veos_vtb
+        echo aaa > ansible/password.txt
+        parent_dir="/data/sonic-mgmt"
+        docker exec sonic-mgmt-masic bash -c "find / -name ansible"
```
Comment (vaibhavhd) [inline tests/pipelines/multi_asic_201911.yml:59]: What's the purpose of this step?

**Review by vaibhavhd — APPROVED** [review]
LGTM. Minor comment added. Please update the description with a link to the successful execution of the pipeline, if possible.

File: tests/pipelines/multi_asic_201911.yml (hunk 1 above)
Comment (SuvarnaMeenakshi) [reply tests/pipelines/multi_asic_201911.yml]: Was not required, removed this.
I have added additional step to reboot the dut before starting the tests.
The multi-asic VS DUT seems to be stable after a reboot, so have added this which could be removed in future once the DUT is stable.

Comment (SuvarnaMeenakshi) [reply tests/pipelines/multi_asic_201911.yml]: sample test result: https://dev.azure.com/mssonic/build/_build/results?buildId=11746&view=logs&j=9edfe4bb-6ff9-5d54-84a0-38181090e645&t=e8814286-d068-5824-9202-504d4c05448a&l=635

File: tests/pipelines/multi_asic_201911.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    type: github
+    name: Azure/sonic-mgmt
+    endpoint: build
+
+stages:
+- stage: Test
+  pool: sonictest
+  variables:
+  - name: dut
+    value: vlab-08
```
Comment (vaibhavhd) [inline tests/pipelines/multi_asic_201911.yml:26]: These hardcoded variables can be made as input parameters with default value. That will help changing the parameters if testbed.csv or any inventory changes.

File: tests/pipelines/multi_asic_201911.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+  jobs:
+  - job: four_asic_vs_test 
+    displayName: "kvmmultiasictest"
+    timeoutInMinutes: 0
+    steps:
+
+    - script: |
+        sudo mkdir -p /data/sonic-vm/images
+        wget https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-multiasic-vs-image-201911/lastSuccessfulBuild/artifact/target/sonic-vs.img.gz
```
Comment (vaibhavhd) [inline tests/pipelines/multi_asic_201911.yml:46]: This image path can be made a variable, that can be defined at the scheduling pipeline stage.

**Review by vaibhavhd — APPROVED** [review]
LGTM, thanks!

File: tests/pipelines/multi_asic_201911.yml (hunk 2 above)
Comment (vaibhavhd) [reply tests/pipelines/multi_asic_201911.yml:26]: This can be addressed in later PR

Comment (SuvarnaMeenakshi) [reply tests/pipelines/multi_asic_201911.yml]: Modified as per suggestion.

File: tests/pipelines/multi_asic_201911.yml (hunk 3 above)
Comment (SuvarnaMeenakshi) [reply tests/pipelines/multi_asic_201911.yml]: Modified as per suggestion.

**Comment by lguohan** [comment]
these are not correct, folks. please move to .azure-pipeline, and consider reuse.

File: tests/pipelines/multi_asic_201911.yml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+stages:
+- stage: Test
+  pool: sonictest
+  variables:
+  - name: inventory
+    value: veos_vtb
+  - name: testbed_file
+    value: vtestbed.csv
+  - name: image_file_path
+    value: https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/buildimage-multiasic-vs-image-201911/lastSuccessfulBuild/artifact/target/sonic-vs.img.gz
```
Comment (lguohan) [inline tests/pipelines/multi_asic_201911.yml:44]: jenkins is going to be deprecated. please download the artifacts from azp to avoid disruption.

Comment (SuvarnaMeenakshi) [reply tests/pipelines/multi_asic_201911.yml:44]: Sure, will add azp job to build multi-asic vs image and modify script to use the right image.

**Pull request merged**